File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell023.cs
using Xunit;

public class TestSmell023
{
    [Fact]
    public void ResetOctaveTest()
    {
        Sound sound = new Sound();
        Assert.Equal(5, sound.GetOctave());

        sound.IncOctave();
        sound.ResetOctave();
        Assert.Equal(5, sound.GetOctave());
    }
}

File: single-language-analysis/dataset-files/java/TestSmell023.java
// Original URL: https://github.com/gabrielseibel1/MusicSeibuilder/blob/e931ba43f8edab3ec38006ef8476e77425726952/SoundTest.java#L15-L23

public class TestSmell023 {
    @Test
    public void resetOctaveTest() throws Exception {
        Sound sound = new Sound();
        assertEquals(5, sound.getOctave());

        sound.incOctave();
        sound.resetOctave();
        assertEquals(5, sound.getOctave());
    }
}
